src/sentry/integrations/jira/integration.py

```diff
@@ -377,21 +380,25 @@ def get_issue_type_meta(self, issue_type, meta):
 
     def get_create_issue_config(self, group, **kwargs):
         fields = super(JiraIntegration, self).get_create_issue_config(group, **kwargs)
-        params = kwargs.get('params', {})
-
-        # TODO(jess): update if we allow saving a default project key
 
+        params = kwargs.get('params', {})
+        default_project = self.get_project_defaults(group.project_id).get('project')
         client = self.get_client()
+
         try:
-            resp = client.get_create_meta(params.get('project'))
+            resp = client.get_create_meta(default_project)
```

Comment:
we don't want to do this. since the jira create issue form is dynamic, we rely on passing the project through the request params. you only want to use default_project if project isn't supplied

Reply:
Oh thank you for telling me. I didn't realize it was dynamic. I'll fix that.